spanner/api/Spanner.Samples.Tests/SpannerFixture.cs

```diff
@@ -220,4 +220,50 @@ public async Task RefillMarketingBudgetsAsync(int firstAlbumBudget, int secondAl
             await cmd.ExecuteNonQueryAsync();
         }
     }
+
+    public async Task CreateVenuesTableAndInsertDataAsync()
+    {
+        string connectionString = $"Data Source=projects/{ProjectId}/instances/{InstanceId}/" +
+            $"databases/{DatabaseId}";
+        // Create connection to Cloud Spanner.
+        using (var connection = new SpannerConnection(connectionString))
```

Comment:
nit: (and remove brackets and indent)
```suggestion
        using var connection = new SpannerConnection(connectionString);
```